src/controllers/proposal.controller.ts

```diff
@@ -9,17 +9,20 @@ import {
   NotFoundException,
 } from '@nestjs/common';
 import { ProposalService } from '../services/proposal.service';
-import { NewProposalDto, UpdateProposalDto } from '../dtos/proposal.dto';
+import {
+  createdProposalDto as NewProposalDto,
```

Comment:
shouldn't this be `createProposalDto`
```suggestion
  createProposalDto as NewProposalDto,
```